shared/src/msi-installer/shared/Files.wxs

```diff
@@ -37,5 +37,27 @@
       </Component>
     </ComponentGroup>
 
+    <?if $(var.Win64) = yes ?>
+    <ComponentGroup Id="Shared.Files.Libdatadog.64" Directory="INSTALLFOLDER.win_x64">
+      <Component Win64="yes">
+        <File Id="libdatadog"
+              Source="$(var.MonitoringHomeDirectory)\win-x64\datadog_profiling_ffi.dll"
```

Comment:
Is this only ever going to be the profiling ffi dll? If so, would it make more sense for this to be in the profiler definition .wxs file, rather than in shared? And if it's going to be _more_ than the profiling_ffi, does it make sense for the file to be called something different? e.g. libdatadog.ffi.dll or something?

Reply:
There is plan from libdatadog repo to rename this libdatadog_profiling.dll.

Reply:
Any progress on that renaming? And to clarify, is this _only_ an ffi for the profiling functionality? Will we have additional dlls for other ffi (for data pipelin etc)

Reply:
not yet.
it's a ffi for profiler/data-pipeline/crash-tracking/... (whatever we decide to have in it)
> Will we have additional dlls for other ffi (for data pipelin etc)

Nop, the goal is to have only one